src/main.ts

```diff
@@ -47,6 +47,29 @@ async function getTrivyEnvVariables(): Promise<{ [key: string]: string }> {
         throw new Error(util.format("Could not download whitelist file. Error: %s", error.message));
     }
 
+    const severityThreshold = core.getInput("severity-threshold");
+    if(severityThreshold) {
+        switch(severityThreshold.toUpperCase()) {
+            case 'UNKNOWN':
+                trivyEnv["TRIVY_SEVERITY"] = "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL";
```

Comment:
Should "Unknown" be everything or just critical ?

Reply:
UNKNOWN is a severity value for vulnerabilities predefined in trivy itself, just like LOW, MEDIUM, HIGH and CRITICAL. If we put UNKNOWN at the bottom in severity order, then we should show all the vulnerabilities which are equal to or above UNKNOWN severity threshold

Reply:
May be we can change the value to 'ALL' or something other than 'UNKNOWN' to make it more understandable?

Reply:
'ALL' makes sense but users might be used-to to the Severity as defined in CVEs. I am in favour of keeping it in sync with that.